packages/sdk/app-framework/src/plugins/common/file.ts

```diff
@@ -17,7 +19,7 @@ export type FileInfo = {
   cid?: string; // TODO(burdon): Meta key? Or other common properties with other file management system? (e.g., WNFS).
 };
 
-export type FileUploader = (file: File) => Promise<FileInfo | undefined>;
+export type FileUploader = (file: File, space: Space) => Promise<FileInfo | undefined>;
 
```

Comment:
It seems weird to me that this type lives in the `AppFramework`. CC @wittjosiah.

It should be fine for now.

Reply:
Yeah, its fine for now. We've defined provides types for "common" plugins that we expect there are likely to be multiple implementations of here for now as an easy place to share types without creating new packages.